src/coreclr/jit/hwintrinsiclistxarch.h

```diff
@@ -817,6 +827,7 @@ HARDWARE_INTRINSIC(AVX512F,         DuplicateEvenIndexed,
 HARDWARE_INTRINSIC(AVX512F,         DuplicateOddIndexed,                        64,              1,     {INS_invalid,           INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_movshdup,           INS_invalid},           HW_Category_SimpleSIMD,             HW_Flag_NoRMWSemantics)
 HARDWARE_INTRINSIC(AVX512F,         ExtractVector128,                           64,              2,     {INS_vextracti128,      INS_vextracti128,       INS_vextracti128,       INS_vextracti128,       INS_vextracti128,       INS_vextracti128,       INS_vextracti128,       INS_vextracti128,       INS_vextractf128,       INS_vextractf128},      HW_Category_IMM,                    HW_Flag_FullRangeIMM)
 HARDWARE_INTRINSIC(AVX512F,         ExtractVector256,                           64,              2,     {INS_vextracti64x4,     INS_vextracti64x4,      INS_vextracti64x4,      INS_vextracti64x4,      INS_vextracti64x4,      INS_vextracti64x4,      INS_vextracti64x4,      INS_vextracti64x4,      INS_vextractf64x4,      INS_vextractf64x4},     HW_Category_IMM,                    HW_Flag_FullRangeIMM)
+HARDWARE_INTRINSIC(AVX512F,         Floor,                                      64,              1,     {INS_invalid,           INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_roundps,            INS_roundpd},           HW_Category_SimpleSIMD,             HW_Flag_NoRMWSemantics|HW_Flag_NoEvexSemantics)
```

Comment:
Avx512 instructions do support EVEX
```suggestion
HARDWARE_INTRINSIC(AVX512F,         Floor,                                      64,              1,     {INS_invalid,           INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_roundps,            INS_roundpd},           HW_Category_SimpleSIMD,             HW_Flag_NoRMWSemantics)
```

We probably need to tweak things a bit in `instrsxarch.h`. Right now `roundps`/`roundpd` are marked as not supporting EVEX. They do actually, in that they become the `vrndscaleps`/`vrndscalepd` instructions and the upper 4-bits of the immediate can now be non-zero.

That also means we probably need to update `instr.cpp` to ensure that `roundps`/`roundpd` are emitted as `vrndscale` in the disassembly

Reply:
makes sense. Will remove the flag.